cmd/clusterctl/client/config/providers_client.go

```diff
@@ -68,6 +68,7 @@ const (
 	CoxEdgeProviderName        = "coxedge"
 	ProxmoxProviderName        = "proxmox"
 	K0smotronProviderName      = "k0sproject-k0smotron"
+	IonosCloudProviderName     = "ionoscloud"
```

Comment:
```suggestion
	IonosCloudProviderName     = "ionoscloud-ionoscloud"
```

As per https://main.cluster-api.sigs.k8s.io/clusterctl/provider-contract.html#adding-a-provider-to-clusterctl

Reply:
Oh my bad